tests/src/main.rs

```diff
@@ -329,6 +327,10 @@ fn rust_flags(codegen_backend_path: &Path) -> String {
         "-Cembed-bitcode=no",
         &format!("-Ctarget-feature=+{}", target_features.join(",+")),
         "-Csymbol-mangling-version=v0",
+        "-Zcrate-attr=no_std",
+        "-Zcrate-attr=feature(register_tool)",
+        "-Zcrate-attr=register_tool(rust_gpu)",
+        "-Zcrate-attr=allow(unused_attributes)",
```

Comment:
The problem here is `spirv-std`, which is `no-std` always. I can make it a `cfg-attr` for `spirv` only. But some random deps might also have `no-std` (`glam` did, for example), that's why I added the `unused_attributes` because the warnings got pretty annoying and you can't always solve them yourself.